src/index.html

```diff
@@ -111,7 +111,7 @@ <h1 class="w-full my-2 text-4xl font-bold leading-tight text-center text-gray-80
       <div
         class="cursor-pointer  bg-white flex-1 rounded-md shadow focus:outline-none focus:shadow-outline transform transition hover:scale-105 hover:shadow-lg duration-300 ease-in-out p-4">
         <div class="flex items-center justify-start overflow-hidden">
-          <img alt="simplilearn" class="bg-gray-50 p-5 rounded-md" src="./static/img/simplilearn.png" />
+          <img alt="simplilearn" class="bg-gray-50 p-5 rounded-md" src="https://github.com/web3community/web3community.github.io/tree/fix_simplelilearn_logo" />
```

Comment:
Wait? How does this work?

Reply:
> Wait? How does this work?

I fixed this in another commit.

Reply:
so why did you make a PR that does nothing and merge it? 😅

Reply:
> so why did you make a PR that does nothing and merge it? 😅

Because I didn't notice that bug.